### PR TITLE
ui/15498584-table-filtering

### PR DESCRIPTION
**Pull request by krawczykkatarzyna**
create component and storybook for table filtering and 

Screen Shot 2023-06-21 at 13 40 39

### CODE REVIEW

File: src/components/common/Extensions/TableFiltering/TableFiltering.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  const { formatMessage } = useIntl();
+
+  return (
+    <div
+      className={`${className} bg-blue-100 py-2 px-3 rounded-lg flex items-center gap-1`}
+    >
+      <div className="text-blue-400 font-semibold text-xs capitalize">
+        {filterType}:
+      </div>
+
```
Comment (KatarzynaStefanowiczPagepro) [inline src/components/common/Extensions/TableFiltering/TableFiltering.tsx:24]: Remove blank line

File: src/components/common/Extensions/TableFiltering/TableFiltering.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      {Array.isArray(filterOptions) ? (
+        filterOptions.map((name) => (
+          <div className="text-blue-400 font-medium text-xs capitalize">
```
Comment (KatarzynaStefanowiczPagepro) [inline src/components/common/Extensions/TableFiltering/TableFiltering.tsx:27]: It should be paragraph

Comment (KatarzynaStefanowiczPagepro) [reply src/components/common/Extensions/TableFiltering/TableFiltering.tsx:27]: We have the class to `font-medium text-xs` main.global.css

```
  .text-4 {
    @apply text-xs font-medium;
  }
```

Please use it

File: src/components/common/Extensions/TableFiltering/TableFiltering.tsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+            {name}
+          </div>
+        ))
+      ) : (
+        <div className="text-blue-400 font-medium text-xs capitalize">
```
Comment (KatarzynaStefanowiczPagepro) [inline src/components/common/Extensions/TableFiltering/TableFiltering.tsx:32]: The same as above

File: src/stories/common/TableFiltering.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,24 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import TableFiltering from '~common/Extensions/TableFiltering/TableFiltering';
```
Comment (KatarzynaStefanowiczPagepro) [inline src/stories/common/TableFiltering.stories.tsx:2]: And a blank line between import packages and other imports.

Also, please correct the import:
`import TableFiltering from '~common/Extensions/TableFiltering/';`

File: src/components/common/Extensions/TableFiltering/TableFiltering.tsx [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+      {Array.isArray(filterOptions) ? (
+        filterOptions.map((name) => (
+          <p className="text-blue-400 text-4 capitalize">{name}</p>
+        ))
+      ) : (
+        <p className="text-blue-400 text-4 capitalize">{filterOptions}</p>
+      )}
+      <button
+        type="button"
+        aria-label={formatMessage({ id: 'handle.close.filter' })}
```
Comment (CzarekDryl) [inline src/components/common/Extensions/TableFiltering/TableFiltering.tsx:33]: I think we are adding `ariaLabel` prefix to aria-labels ->
image

**Review by CzarekDryl — APPROVED** [review]

**Pull request merged**